src/pages/advertiser/screens/Advertiser/Advertiser.tsx

```diff
@@ -57,7 +60,7 @@ const Advertiser = () => {
                 onClickUnblock={() => showModal('BlockUnblockUserModal')}
                 setShowOverlay={setShowOverlay}
             >
-                <ProfileContent id={id} setAdvertiserName={setAdvertiserName} setShowOverlay={setShowOverlay} />
+                <ProfileContent data={data} setAdvertiserName={setAdvertiserName} setShowOverlay={setShowOverlay} />
```

Comment:
doesn't this break the Profilecontent in myprofile and myprofilemobile

Reply:
already fixed it for both cases 👍